internal/creds_test.go

```diff
@@ -120,6 +120,34 @@ func TestJWTWithScope(t *testing.T) {
 	}
 }
 
+func TestJWTWithScopeAndUniverseDomain(t *testing.T) {
```

Comment:
should we test invalid json does yield an error?  We're really just propagating json.Unmarshal's error up so it's not a particularly useful case.

Reply:
Hmm, what do you suggest doing instead?